docs/includes/architecture.md

```diff
@@ -2,7 +2,7 @@ The nio Platform is built for users like you to build real-world, innovative, an
 
 Every state, every change, and every interaction in our world is a signal—from the percentage of moisture in the soil beneath your feet, to the vibration of your car motor, to the individual characters within your last tweet. Every _thing_ is producing context that already has—or will soon—become digitally available in the Internet of Things era. But how can these trillions of disparate signals work harmoniously to produce any meaningful value?
 
-nio was built to act as the nervous system for a world with trillions of unique [signals](https://docs.n.io/services/service-design-patterns/signal-structure.md). It  allows you to create real-world, innovative, and impactful solutions with connected, distributed systems.
+nio was built to act as the nervous system for a world with trillions of unique [signals](/services/service-design-patterns/signal-structure.md). It  allows you to create real-world, innovative, and impactful solutions with connected, distributed systems.
```

Comment:
good catch, but this file is unique because workshops also uses it…so we can't use internal links 😞 they have to be full, external links to work (unless you know a trick). I think the problem then (if it uses https://docs.n.io) is that it needs to link to .html instead of .md. I know I told you .md, but for external… .html

Reply:
There should be no need to share documentation content between workshops and docs. If it's the same content it can exist in both places but I would question why we have the same content multiple places. If it belongs in documentation (this content seems to) then we can link to it from workshops, it doesn't need to be included there.

Reply:
Ease of maintenance is the goal (why we don't duplicate it in both places)—but it's good to question why we have the same content in both places. In this case, I think it is because it is a relevant overview. We change the ending in workshops to drive people to the first tutorial. This content started in workshops and has moved over to docs. I'll discuss with the dog-food team to see if we can't come up with something better.